vit-servicing-station-cli/src/csv/loaders.rs

```diff
@@ -0,0 +1,122 @@
+use crate::{db_utils::db_file_exists, task::ExecTask};
+use csv::Trim;
+use serde::de::DeserializeOwned;
+use std::io;
+use structopt::StructOpt;
+use vit_servicing_station_lib::db::{
+    load_db_connection_pool, models::funds::Fund, models::proposals::Proposal,
+    models::voteplans::Voteplan,
+};
+
+#[derive(Debug, PartialEq, StructOpt)]
+pub enum CSVDataCmd {
+    /// Add provided tokens to database. If --tokens is not provided the binary will read them from the `stdin`
+    Dump {
```

Comment:
This should be `Load` ans an action since it loads from csv -> db.
We may also imlement `Dump` in the furture from db -> csv